src/common/devcard.ts

```diff
@@ -82,18 +84,23 @@ export async function getDevCardDataV1(
   const start = subYears(now, 1).toISOString();
   const end = now.toISOString();
   const user = await con.getRepository(User).findOneByOrFail({ id: userId });
-  const [articlesRead, tags, sources, rank] = await Promise.all([
+  const [articlesRead, tags, sources, rank, streak] = await Promise.all([
     con.getRepository(ActiveView).countBy({ userId }),
     getMostReadTags(con, { userId, limit: 4, dateRange: { start, end } }),
     getFavoriteSources(con, userId),
     getUserReadingRank(con, userId, user?.timezone, 2),
+    await con.getRepository(UserStreak).findOneOrFail({
+      where: { userId },
+      select: ['maxStreak'],
+    }),
   ]);
   return {
     user,
     articlesRead,
     tags,
     sourcesLogos: sources.map((source) => source.image),
     rank,
+    longestStreak: streak?.maxStreak ?? 0,
```

Comment:
Can we just make it `maxStreak` as in streak column to avoid confusion in naming?

Reply:
Sure

Reply:
On that note, we can actually avoid making any changes here, we can just add the existing `userStreak` query to the same call.

Reply:
Yeah, good idea @sshanzel, I agree 👍